src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/ExplicitInitializationCheck.java

```diff
@@ -71,7 +74,17 @@
         return new int[] {TokenTypes.VARIABLE_DEF};
     }
 
+    /**
+     * Sets whether only explicit initialization made to null should be checked.
+     * @param onlyObjectReferences whether only explicit initialization made to null
+     *                             should be checked
+     */
+    public void setOnlyObjectReferences(boolean onlyObjectReferences) {
+        this.onlyObjectReferences = onlyObjectReferences;
+    }
+
     @Override
+    @SuppressWarnings("checkstyle:cyclomaticcomplexity")
```

Comment:
We prefer comment suppressions instead.
Ex: `// -@cs[CheckName] Reason for suppression`